biosys/apps/main/api/views.py

```diff
@@ -212,9 +213,68 @@ def perform_update(self, serializer):
 
 
 class FormViewSet(viewsets.ReadOnlyModelViewSet):
-    permission_classes = (IsAuthenticated, DRYPermissions)
+    permission_classes = (IsAuthenticated,)
     serializer_class = serializers.FormSerializer
     queryset = models.Form.objects.all().distinct()
+    filter_fields = ('id', 'name', 'dataset', 'dataset__project', 'dataset__name',)
+
+
+@api_view()
+def form_structure(request):
```

Comment:
Could you put in comment an example of the json this view returns? At least is basic structure. Or better create a unit test that test the json.